content/hardware/07.opta/opta-family/opta-blues-exp/product.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Wireless for Arduino Opta
+url_shop: https://shop.blues.com/products/wireless-for-plc
+url_guide: /tutorials/opta-analog-exp/user-manual
```

Comment:
Are we using the analog expansion user manual? 🤔

Reply:
Thanks for noticing it, our docs page is not using anymore the `url_guide` metadata parameter. Removed

Reply:
No problem, didnt know 👌